### PR TITLE
Blog post: Image Analysis

### PR DESCRIPTION
**Pull request by PrimozGodec**

### CODE REVIEW

File: content/blog/2022/2022-02-11-edit-image-collection.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+### Training the model to sort images
+
+The Image Grid is excellent to discover similar images. Still, it is not practical to sort photos into category groups. If we would like to do that more often, we would always need to assign images to groups manually. Can we have a pipeline that would allow us to make a model that we can reuse later when we take new photos s?
+
+**Related**: {{< link_new url="https://www.youtube.com/watch?v=lvgx62a8XQk&list=PLmNPvQr9Tf-aRstY69vGAPO_c5RaceBhN&index=1&ab_channel=OrangeDataMining" name="Video on image classification">}}
+
+We can train a classification model that will mark images with categories. 
+
+We prepared a separate training dataset with pictures in 5 subfolders. Folders are named: city, food, mountains, sunsets and winter. Names of the flowers define categories; once the model is trained, it will always classify images in those categories.
```
Comment (ajdapretnar) [inline content/blog/2022/2022-02-11-edit-image-collection.md:69]: Names of the flowers?

File: content/blog/2022/2022-02-11-edit-image-collection.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+We also connected [the Save Model widget](/widget-catalog/model/savemodel) to the Logistic Regression. This way, the model is saved for later and can be reused. 
+
+### Want to learn more?
+
+We showed two ways of analysing images with Orange's Image Analytics module. The Image Grid is a great tool to identify similar images and see them in the grid. The second one needs an extra effort to prepare a training dataset, but this leads to an automatic pipeline to identify categories.
+
+If you want to learn more about Image Analysis in [Orange watch the tutorial wideos](https://youtube.com/playlist?list=PLmNPvQr9Tf-aRstY69vGAPO_c5RaceBhN) or read related blogs: [Image Analytics: Clustering](/blog/2017/04/03/image-analytics-clustering/]), [Clustering of Monet and Manet](/blog/2018/05/09/clustering-of-monet-and-manet/), and [Outliers in Traffic Signs](/blog/2017/04/25/outliers-in-traffic-signs/)
```
Comment (ajdapretnar) [inline content/blog/2022/2022-02-11-edit-image-collection.md:91]: [Orange watch the tutorial wideos] --> Orange [watch the tutorial videos]
(Orange should probably not be included in the link?)

File: content/blog/2022/2022-02-11-edit-image-collection.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+This blog post will show two approaches for identifying categories and sorting images. The first one is unsupervised, and the second one is supervised. 
+
+First, we load photos. We load a folder of images with a collection that we would like to sort in the subfolders ([downlaod dataset](https://github.com/PrimozGodec/datasets/blob/master/image-analytics-collection/test-photos.zip)).
+
+{{< window-screenshot src="/blog_img/2022/2022-02-11-inspect-images.jpg" >}}
+
+Both approaches will include embedding images. We connect the [Image Embedding](/widget-catalog/image-analytics/imageembedding/) widget to the Import Images widget to embed pictures and select **Inception v3 embedder**.
+
+*Our server delete all images immediately after embedding. If you would not like to share your photos with our server anyway, you can select the SqueezeNet embedder that embeds images at your computer.*
```
Comment (ajdapretnar) [inline content/blog/2022/2022-02-11-edit-image-collection.md:41]: delete --> deletes

File: content/blog/2022/2022-02-11-edit-image-collection.md (hunk 1 above)
Comment (PrimozGodec) [reply content/blog/2022/2022-02-11-edit-image-collection.md]: Haha. This one is probably Grammarly's correction that I didn't notice.

**Pull request merged**